index.html

```diff
@@ -2,295 +2,317 @@
-              <div class="skill-container">
-                <h3 class="skill-title">Languages</h3>
-                <button class="btn-chev" type="button">
-                  <img src="image/chev-dow.png" alt="" />
-                </button>
-              </div>
-              <div class="separator-line"></div>
-              <ul class="language-list">
-                <li>
-                  <img src="image/javascript.png" alt="JavaScript" />JavaScript</li>
-                <li>
-                  <img src="image/html.png" alt="HTML" />HTML</li>
-                <li>
-                  <img src="image/css.png" alt="CSS" />CSS</li>
-              </ul>
-            <!-- /languages -->
-            <li class="my-skill">
-              <div class="skill-container">
-                <h3 class="skill-title">Frameworks</h3>
-                <button class="btn-chev" type="button">
-                  <img src="image/chev-right.png" alt="" />
-                </button>
-              </div>
-              <div class="separator-line"></div>
+        </div>
+        <div class="about-skills">
+          <div class="skills about-skill">
+            <div class="icon">
+              <img src="images/Icons(1).png" alt="" />
+            </div>
+            <h4>Skills</h4>
+            <ul>
+              <li>CodeKit</li>
+              <li>GitHub</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+        </div>
+      </div>
+    </section>
+    <!-- Contact Section -->
+    <section class="contact-section">
+      <div class="container-contact">
+        <div class="container-contact2">
+          <h2 class="contact-description">
+            I am interested in knowing about new projects, so if you would like to communicate please get in touch with me.
+        </h2>
+        <form action="https://formspree.io/f/xayknkna" method="POST">
+          <div class="name-form">
+            <input
+            type="text"
+            name="firstName"
+            id="firstName"
+            placeholder="First Name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="text"
+            name="lastName"
+            id="lastName"
+            placeholder="Last Name"
+            maxlength="30"
+            required
+          />
+          </div>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            placeholder="Email address"
+            required
+          />
+
+          <textarea
+            name="comment"
+            placeholder="Enter text here"
+            id="comment"
+            cols="30"
+            rows="10"
+            maxlength="500"
+          ></textarea>
```

Comment:
- [ ] kindly align all of your inputs together , currently the name field is not aligned with the others fields

![image](https://user-images.githubusercontent.com/89282221/187993666-cf50c27e-7785-4d66-ae49-82384bf94074.png)

Reply:
The "name-form" class is under "Display: none"... But I included that in the property definition instead of "name". I changed that and it is ok. The changed code is now 
**form > input[type="text"],**
  form > input[type="email"],
  form > textarea {
    width: 100%;
    padding: 15px;
    border: 1px solid #e0e0e0;
    border-radius: 4px;
    margin: 10px;
    font-size: 14px;
    color: #172b4d;
  }